### PR TITLE
fixing : Translation not working in catalog-add-to-cart.js

### PR DESCRIPTION
**Pull request by septa-setiawan**
### Description

Fixing : Translation not working in catalog-add-to-cart.js

### Fixed Issues (if relevant)

1. magento/magento2#14024: Translation not working in catalog-add-to-cart.js
2. ...

### Manual testing scenarios

1. Adding the translation in app/i18n/<vendor>/en_GB.csv
2. Apply changes in app/code/Magento/Catalog/view/frontend/web/js/catalog-add-to-cart.js
3. Deploy then flush cache

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @septa-setiawan. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=17381) <br/>All committers have signed the CLA.

File: app/code/Magento/Catalog/view/frontend/web/js/catalog-add-to-cart.js [hunk 1]
```diff
@@ -188,8 +188,8 @@ define([
                 var addToCartButtonTextDefault = self.options.addToCartButtonTextDefault || $t('Add to Cart');
 
                 addToCartButton.removeClass(self.options.addToCartButtonDisabledClass);
-                addToCartButton.find('span').text(addToCartButtonTextDefault);
-                addToCartButton.attr('title', addToCartButtonTextDefault);
```
Comment (sidolov) [inline app/code/Magento/Catalog/view/frontend/web/js/catalog-add-to-cart.js:192]: It' not a good idea to remove the extension point for button title, the title may be redefined by 3d party extension via `self.options.addToCartButtonTextDefault` and will be broken after such changes

**Review by sidolov — CHANGES_REQUESTED** [review]

File: app/code/Magento/Catalog/view/frontend/web/js/catalog-add-to-cart.js (hunk 1 above)
Comment (septa-setiawan) [reply app/code/Magento/Catalog/view/frontend/web/js/catalog-add-to-cart.js:192]: OK will trying another way, thanks.

**Comment by Karlasa** [comment]
@septa-setiawan  Are you sure it's even broken in latest 2.2 release?

**Comment by sidolov** [comment]
Hi @septa-setiawan , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**